ops/opset13/opset13_test.go

```diff
@@ -33,16 +33,16 @@ func TestGetOperator(t *testing.T) {
 			newAdd(),
 			nil,
 		},
-    {
-      "Atan",
-      newAtan(),
-      nil,
-    },
-    {
-      "Atanh",
-      newAtanh(),
-      nil,
-    },
+		{
+			"Atan",
```

Comment:
Are these changes belonging here?

Reply:
somehow they were not formatted right, so I'd say it is nice to format them right and we can include it here.

Reply:
However, the formatting was already fixed in another merge request, so the changes are not needed anymore.